test/bob/pipeline/core_test.clj

```diff
@@ -303,3 +303,47 @@
               (-> @(logs-of "dev" "test" 1 0 100)
                   :body
                   :message))))))
+
+(deftest get-pipeleines
+  (testing "Filter pipelines"
+    (with-redefs-fn {#'db/get-pipelines           (fn [& args] [{:name "test:Test" :image "test 1.7"}])
```

Comment:
can you assert that the mocked functions are called with the right params like https://github.com/bob-cd/bob/blob/master/test/bob/pipeline/internals_test.clj#L59